staging/src/k8s.io/api/extensions/v1beta1/generated.proto

```diff
@@ -847,6 +847,12 @@ message PodSecurityPolicySpec {
   // runAsUser is the strategy that will dictate the allowable RunAsUser values that may be set.
   optional RunAsUserStrategyOptions runAsUser = 11;
 
+  // RunAsGroup is the strategy that will dictate the allowable RunAsGroup values that may be set.
+  // If this field is omitted, RunAsGroup can take any value. This field requires the RunAsGroup
+  // alpha feature gate to be enabled.
+  // +optional
+  optional RunAsGroupStrategyOptions runAsGroup = 22;
+
```

Comment:
@liggitt This version of PSP is deprecated (moved to the policy API group). Do we still need to add new fields here, or can we let this version fall behind?

Reply:
we need to keep field parity until we are sure we will never have to convert from policy/v1beta1 to extensions/v1beta1. That will occur once we remove the ability to influence etcd storage versions and stop serving PSPs under the extensions/v1beta1 API group. That may happen in 1.13, but I'm not prepared to commit to that at this point.